templates/index.html

```diff
@@ -32,7 +32,7 @@
          border="0"
          alt="Tracking pixel" />
   </noscript>
-  <div class=" p-navigation__overlay">
+  <div class="p-navigation__overlay is-dark">
```

Comment:
Ideally this should be on body.

Not sure what other styling is current `is-graphite` class on body have, but I think it would be best to remove it and use official is-dark themeing from Vanilla.

Reply:
Thanks, I moved `is-dark` to body. I think `is-graphite` was previously hardcoding dark colours but the new theme should support that now.